app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -79,9 +81,27 @@ class PlatformInitiativesService @Inject()(
         artefact              = "auth-client",
         version               = Version(5,6,0,"5.6.0"),
         team                  = team
+      ),
+      createUpgradeInitiative(
+        initiativeName        = "Scala 2.12 Upgrade",
+        initiativeDescription = s"Scala 2.12 upgrade - Deprecate [Scala 2.11 and below](" + url"https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=com.typesafe.play&artefact=play&team=$teamName&flag=production&scope=compile&versionRange=[0.0.0,2.8.0)&asCsv=false".toString.replace(")", "\\)") + " ) | [Confluence](" + url"https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=275944511" + ").",
+        group                 = "org.scala-lang",
+        artefact              = "scala-library",
+        version               = Version(2,12,0,"2.12.0"),
+        team                  = team,
+        experimental          = displayExperimentalInitiatives
```

Comment:
should we set `experimental = true` and then if  `displayExperimentalInitiatives = false`, we filter them out? Rather than changing whether we classify these initiatives as experimental?